lib/index.js

```diff
@@ -70,15 +70,18 @@ const plugin = {
 
 plugin.configs.recommended = [
     {
+        name: "markdown/plugin",
         plugins: {
             markdown: plugin
         }
     },
     {
+        name: "markdown/processor",
         files: ["**/*.md"],
         processor: "markdown/markdown"
     },
     {
+        name: "markdown/rules/recommended",
```

Comment:
```suggestion
        name: "markdown/recommended",
```

Reply:
Other plugins just use "recommended" for this name.

Reply:
> Other plugins just use "recommended" for this name.

Resolved.